cockroachcloud/cluster-management.md

```diff
@@ -137,6 +137,8 @@ You can add or remove up to nine regions at a time through the Console. Note tha
 
 ### Remove a region from your cluster
 
+When you remove a region from your cluster, the node in that region with the highest ordinal will be decommissioned first. Any ranges on that node will be up-replicated to other nodes, and once decommission is complete that node will be shut down. This process is then repeated for every other node in the region. To remove a region from your cluster:
```

Comment:
suggest making 'decommissioned' a link to https://www.cockroachlabs.com/docs/v22.2/node-shutdown?filters=decommission#decommission-the-node to offer more detail, if wanted

'up-replicated' could be a link to these Replication Dashboard docs that show up-replication after decommission: https://www.cockroachlabs.com/docs/stable/ui-replication-dashboard.html#snapshot-data-received 

finally the fact that we can remove a region implies more than one region, which implies it is a multi-region cluster.  Suggest a slight modification to the text to say something like "when you remove a region from a [multi-region cluster](https://www.cockroachlabs.com/docs/cockroachcloud/plan-your-cluster.html#multi-region-clusters), ..." since due to links and search, a user can land directly on this sub-heading of the page without seeing the context at beginning of parent section